packages/cli/config/schema.ts

```diff
@@ -875,6 +875,15 @@ export const schema = {
 		},
 	},
 
+	enterprise: {
+		features: {
+			sharing: {
+				format: Boolean,
+				default: false,
+			},
+		},
+	},
```

Comment:
Not up to date on the latest UM but just to make sure we won't need to separately enable sharing for creds vs. workflows? Else `credentialSharing`.

Reply:
I don't think we'll be decoupling workflow and credential sharing. so far the plan is that user-management can be enabled without sharing, but when sharing is enabled, it enables both credential sharing, as well as workflow sharing.